client/api/encoder.py

```diff
@@ -0,0 +1,33 @@
+from PIL import Image
+import numpy as np
+import base64
+import io
+
+
+class Encoder:
+
+    def __init__(self) -> None:
+        pass
+
+    @staticmethod
+    def encode(data: np.ndarray):
+
+        image = Image.fromarray(data)
+        output = io.BytesIO()
+        image.save(output, format='JPEG')
```

Comment:
Możliwe, że zmienimy to na zaszyfrowanie `numpy.ndarray`. Nie jestem pewien co będzie wygodniejsze, zobaczymy po połączeniu tych modułów.